docs/client_guide.md

```diff
@@ -241,7 +241,8 @@ Notice: this feature requires the MEMORY_LOCAL_HADOOP mode.
 | Property Name                  | Default | Description                                                             |
 |--------------------------------|---------|-------------------------------------------------------------------------|
 | tez.rss.avoid.recompute.succeeded.task | false   | Whether to avoid recompute succeeded task when node is unhealthy or black-listed |
- 
+| tez.rss.client.max.buffer.size | 3k | The max buffer size in map side |
```

Comment:
What's the difference between this option and `rss.writer.buffer.size`? Could you elaborate this description?

Reply:
ok.

Reply:
rss.client.max.buffer.size  -->  maxSegmentSize -->  This parameter is the size of the WrappedBuffer created
rss.writer.buffer.size      -->  maxBufferSize  -->  When buffer.getDataLength > maxBufferSize, the buffer will be sent to the server
This logic is a reuse of MapReduce.